apache-rat-core/src/main/java/org/apache/rat/configuration/builders/ChildContainerBuilder.java

```diff
@@ -99,11 +102,11 @@ public AbstractBuilder add(Collection<IHeaderMatcher.Builder> children) {
     public List<IHeaderMatcher> getChildren() {
         return children.stream().map(IHeaderMatcher.Builder::build).collect(Collectors.toList());
     }
-    
+
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder(this.getClass().getSimpleName()).append( ":");
-        children.stream().map(Object::toString).forEach( x -> sb.append("\n").append(x));
+        StringBuilder sb = new StringBuilder(this.getClass().getSimpleName()).append(":");
+        children.stream().map(Object::toString).forEach(x -> sb.append("\n").append(x));
```

Comment:
Not sure if it is a problem on Windows to have Linux file endings ... otherwise System.lineSeparator() again.